configuration/pih/htmlforms/ncd-adult-initial.xml

```diff
@@ -120,38 +131,6 @@
                 jq('#disposition select:first').change(function() {
                     jq('#disposition .field-error').first().html('').hide(); // hide any previous error if disposition is changed
                 });
-
```

Comment:
The below was made more generic and moved into the htmlFormFunctions class in the pihemr config